Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -89,13 +105,25 @@ def http_request(client, method, url_suffix, json=None, wait=0, retries=0):
                                                                 r.headers.get("X-Rate-Limit-Reset")))
     if r.status_code != 200:
         if r.status_code == 429:
-            if retries <= 0:
-                # Error in API call to URLScan.io [429] - Too Many Requests
-                return_error('API rate limit reached [%d] - %s.\nUse the retries and wait arguments when submitting '
-                             'multiple URls' % (r.status_code, r.reason))
-            else:
-                time.sleep(wait)  # pylint: disable=sleep-exists
-                return http_request(method, url_suffix, json, wait, retries - 1)
+            try:
+                ScheduledCommand.raise_error_if_not_supported()
+                scheduled_command = ScheduledCommand(
+                    command=demisto.command(),
+                    next_run_in_seconds=60,
+                    args=demisto.args(),
+                    timeout_in_seconds=600
+                )
+                results = CommandResults(scheduled_command=scheduled_command, error_type=ErrorTypes.RATE_LIMITED)
+                demisto.results(results.to_context())
+                return {'RateLimited': True}
+            except DemistoException:
```

Comment:
we need to make sure it is specifically the not supported error.
also, It may be a good idea to call the check directly and not use exception handling for this, because it is not really an error.